### DISCUSSION TITLE
How to apply previously made backup to new docker container running this server?

### DISCUSSION THREAD

**Discussion by 36desires** [thijsvanloef/palworld-server-docker#189] [Q&A]
Hello dear community,

unfortunately I have accidently removed the docker container running my palworld server today trying to access its save-files/server data. I luckily have backed up the data shortly before this happened with the "docker exec palworld-server backup" command.

Unfortunately I am still rather new to using linux and docker and have barely understood how the "volumes" really work. For my Minecraft & Valheim Servers I had specified the volumes a bit differently than stated in this tutorial here and had always received a classic "server / world / map"-folder on my user/home/[game server files].

Now when I want to run a new container with this palworld dedicated server, how do I apply the previously made backup? I don't even know how to find out the exact XXX.tar name of the backup, how to convert it into a world or savefile nor am I able to find out how to access the folders in question properly. 

![image_2024-01-28_201314424](https://github.com/thijsvanloef/palworld-server-docker/assets/148643697/cbcf33e5-338a-4272-8115-fd03728cbffa) 
the first seemed to be my volume (as the other ones were creating since ive tried fixing it) therefore I thought it was this folder "/var/lib/docker/volumes/5f386b548d006a35fe05a4ecc83905950eb1325da4578effa3bae84fb2c32cea" carrying the data but what do I do now? It's all empty and i am losing my mind. 

Thanks already to you all and thijsvanloef for creating this container,

Simon

**Comment by thijsvanloef** [comment]
There is a section of the Readme explaining this: https://github.com/thijsvanloef/palworld-server-docker?tab=readme-ov-file#manually-restore-from-a-backup

**Reply by thijsvanloef** [reply]
you can also copy all of the backups to your local file system using the following command: 
`docker cp palworld-server:/palworld/backups .`
